Jenkinsfile

```diff
@@ -71,6 +71,10 @@ pipeline {
 		buildDiscarder(logRotator(numToKeepStr: '30'))
 	}
 
+	triggers {
```

Comment:
Please do not mix tabs with spaces, especially not in a single line (L76). If we transition to spaces for Jenkinsfile, 1 tab should be 4 spaces. Only transition lines or blocks touched (which is the case).